common/views/components/Picture/Picture.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import type {Picture as PictureProps} from '../../../model/picture';
+import Tasl from '../Tasl/Tasl';
+import {imageSizes} from '../../../utils/image-sizes';
+import {convertImageUri} from '../../../utils/convert-image-uri';
+
+type Props = {|
+  images: PictureProps[],
+  extraClasses?: string
+|};
+
+const Picture = ({ images, extraClasses }: Props) => {
+  const lastImage = images[images.length - 1];
+  return (
+    <figure className='relative no-margin'>
+      <picture className={extraClasses || ''}>
+        {images.map(image => {
+          if (image.width) {
+            const sizes = imageSizes(image.width);
+            return (
+              <source
+                key={image.contentUrl}
+                media={image.minWidth ? `(min-width: ${image.minWidth})` : ''}
+                data-srcset={sizes.map(size => {
+                  return image.contentUrl && `${convertImageUri(image.contentUrl, size, false)} ${size}w`;
+                })} />
+            );
+          }
+        })}
+
+        {lastImage && lastImage.contentUrl && lastImage.width && <img
+          height={lastImage.height}
+          width={lastImage.width}
+          className='image lazy-image lazyload'
```

Comment:
Elsewhere, we conditionally add the `lazyload` class (although I don't think we've ever had an image that we _don't_ want to lazyload).

Maybe (in another PR) we should either just make all images lazyload (without the option to turn it off), or possibly make lazyload the default with the ability to override (which is the opposite of the current behaviour).

Reply:
That was my thought... just made the code harder to read so left it out.
[See here](https://github.com/wellcometrust/wellcomecollection.org/issues/new)